examples/05_example_patchpredictor.ipynb

```diff
@@ -375,7 +375,7 @@
         "\n",
```

Comment:
<p>This cell does not appear to be explained.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TIA-Lab/tiatoolbox/pull/132/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>